docs/design/fabric-v2+/trusted_ledger.md

```diff
@@ -0,0 +1,162 @@
+
+- check config sequence number increased by 1
+- check is authorized Update [source](https://github.com/hyperledger/fabric/blob/f27912f2f419c3b35d2c1df120f19585815eceb0/common/configtx/update.go#L115)
+    - verify ReadSet [source](https://github.com/hyperledger/fabric/blob/f27912f2f419c3b35d2c1df120f19585815eceb0/common/configtx/update.go#L18)
+    - verify DeltaSet [source](https://github.com/hyperledger/fabric/blob/f27912f2f419c3b35d2c1df120f19585815eceb0/common/configtx/update.go#L68)
+        - for each item validate policy [source](https://github.com/hyperledger/fabric/blob/f27912f2f419c3b35d2c1df120f19585815eceb0/common/policies/policy.go#L133)
+
+Lifecycle validation:
+
+
+Default validation:
+
+- Validating identities that signed the transaction
+    - read/write check?
+- Verifying the signatures of the endorsers on the transaction
+    - can endorse?
+- Ensuring the transaction satisfies the endorsement policies of the namespaces of the corresponding chaincodes.
+
+
+## Approach
+
+Restrict Fabric functionality:
+
+- keep Fabric validation changes in sync with TLCC
+- Reduces validation logic to a minimum
+    - minimizes catch-up 
+- Restrict the notion of endorsement policy; 
+- Simplify chaincode versioning (TODO)
+    - No chaincode updates for now
+- Support for default MSP only
+
+
+Any transactions/blocks with unsupported features are ignored/aborted
+
+Defined process to develop TLCC:
+- keep redundant validation code (in stock peer and TLCC) in sync
+- keep code relation traceable and changes trackable
+- consistent naming and code structure as much as possible
+- bonus: 
+    - automated code-changes notification; notifies whenever relevant Fabric code changes and might FPC must be updates.
+
+### Non-features
+
+- No state-based endorsement
+- No custom MSP (no idemix)
+- No support for custom endorsing/validation plugins for non-FPC and FPC chaincodes
+- Authentication and decorators
+- TODO check for more non-features in RFC
+
+## Design
+
+### Chaincode execution support
+    - secure channel
+    - validate proposal
+        - check org is "write"
+        - replay protection
+
+### Ledger maintainance
+
+Channel configuration:
+- Parse channel definition
+    - parse consensus definition
+- parse channel reader/writers
+- validate block signatures (Note that this check implies correctness as onfig blocks are validated by the ordering service ([see code](https://github.com/hyperledger/fabric/blob/f27912f2f419c3b35d2c1df120f19585815eceb0/orderer/common/msgprocessor/standardchannel.go#L131)))
+- Maintain msp metadata for signature validation
+- parse Lifecycle policy 
+    - validate Lifecycle policy
+    We only support: a majority of Org.member (See [Link](https://hyperledger-fabric.readthedocs.io/en/release-2.2/chaincode_lifecycle.html#install-and-define-a-chaincode))
+
+Access control:
+- delegate this "reader/writer/admin" check to admins;
```

Comment:
We wouldn't really delegate the _(authorization) check_ to admins as this text seems to imply?  we would delegate to admins the  _policy check_ that the authorization policy does not allow ruling out an endorsement which otherwise would be a valid endorsement regarding our restrictions as mentioned above?